### PR TITLE
Adding support for SQLite

### LINKED ISSUE #50: Support for SQLite3
I tried to connect to a local SQLite3 database using Locopy and it threw the following error:

```
ValueError: parameters are of unsupported type
```

After doing some digging, it looks like the error is being thrown by the default argument in `locopy.database.Database.execute()` for `params`. Once I started using `params=()`, it started working.

e.g.

```python
with locopy.Database(dbapi=sqlite3, database=':memory:') as cmd:
  cmd.execute('''CREATE TABLE stocks (date text, qty real)''', params=())
```

I tested `params=()` with `locopy.snowflake.Snowflake` as well and it worked.

### LINKED ISSUE #55: Adding a "USE SCHEMA" call for Snowflake
In the `Snowflake` class, I'd like to add a feature for automatically running the command

```sql
USE SCHEMA {{ schema_name }}
```

upon connecting to Snowflake. The package already has a similar feature with `database` and `warehouse`, so I think it would be a relatively simple feature addition.

### PR DESCRIPTION
**Pull request by ak-gupta**
In this pull request I have added support for SQLite as well as an additional feature for Snowflake users. If the user supplies `schema` in their credentials, `locopy.snowflake.Snowflake` will run `USE SCHEMA {schema}` for the user.

Fixes #50 
Fixes #55

### CODE REVIEW

**Comment by fdosani** [comment]
@ak-gupta I think for the failing tests we can change the dev requirements files to the following:

```
psycopg2>=2.7.7
pg8000>=1.13.1
snowflake-connector-python>=1.7.6
```

Just set the connectors to be `>=`

**Comment by ak-gupta** [comment]
Oh, do I need to increment the version before merging to master?

**Comment by fdosani** [comment]
> Oh, do I need to increment the version before merging to master?

No need. I'll take care of that once we are ready to cut a new release.

**Comment by ak-gupta** [comment]
Ok great, just wanted to make sure!

File: locopy/database.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def _disconnect(self):
         else:
             logger.info("No connection to close")
 
-    def execute(self, sql, commit=True, params=None):
+    def execute(self, sql, commit=True, params=()):
```
Comment (jborchma) [inline locopy/database.py]: I am curious about this change here. Is this to align with some convention for SQLite?

Comment (ak-gupta) [reply locopy/database.py]: Yeah, when I tried to use `Database` with `params=None` it didn't work

**Pull request merged**